tests/test-tokenizer-random.py

```diff
@@ -134,15 +150,24 @@ def __init__(self, dir_tokenizer: str):
         self.add_bos_token = getattr(self.model, "add_bos_token", add_bos_token)
         self.add_eos_token = getattr(self.model, "add_eos_token", add_eos_token)
         # build vocab
-        tokens = list(self.model.get_vocab().values())
-        self.vocab = self.model.batch_decode(tokens, skip_special_tokens=True)
-        self.vocab = list(sorted(self.vocab))
+        self.vocab = self.get_vocab(detokenize=True)
         # tokens and lists
         self.special_tokens = list(self.model.all_special_tokens)
         self.added_tokens   = list(self.model.added_tokens_encoder)
         self.bos_token = self.model.bos_token
         self.eos_token = self.model.eos_token
 
+    def get_vocab(self, detokenize=False) -> list[str]:
+        max_token_id = max(self.model.get_vocab().values())
+        if detokenize:
+            ids = list(range(max_token_id + 1))
+            vocab = self.model.batch_decode(ids, skip_special_tokens=False)
```

Comment:
Do you think this should be used in the convert script instead of directly getting the strings from `tokenizer.vocab`?